R/init.R

```diff
@@ -114,9 +114,11 @@ init <- function(data,
                  footer = tags$p(),
                  id = character(0)) {
   logger::log_trace("init initializing teal app with: data ({ class(data)[1] }).")
-  data <- teal.data::to_relational_data(data = data)
 
-  checkmate::assert_class(data, "TealData")
+  if (!inherits(data, c("TealData", "teal_data"))) {
+    data <- teal.data::to_relational_data(data = data)
+  }
+  checkmate::assert_multi_class(data, c("TealData", "teal_data"))
```

Comment:
I don't understand this assertion in the context of the preceding conversion of these classes to `TealData`.

Also, am I right to assume this function will be removed during the final changes?

Reply:
Assertion will be replaced by `checkmate::assert_multi_class(data, c("teal_data", "ddl"))`

Reply:
was thinking about the same